en/docs/Learn/APISecurity/OAuth2DeepDive/token-revocation.md

```diff
@@ -5,29 +5,26 @@
 After issuing an access token, a user or an admin can revoke it in case of theft or a security violation. You can do this by calling the Revoke API using a utility like cURL. The Revoke API's endpoint URL is `http://localhost:8280/revoke` .
 
 !!! note
-    You can also revoke refresh tokens. For more information, see [Revoking a refresh token](../../../../Learn/ConsumeAPI/ManageApplication/GenerateKeys/GrantTypes/refresh-token-grant/#revoking-a-refresh-token) .
+    You can also revoke refresh tokens. For more information, see [Revoking a refresh token](../../../../Learn/APISecurity/OAuth2DeepDive/GrantTypes/refresh-token-grant/#revoking-a-refresh-token).
```

Comment:
let's use {{base_path}} instead of relative paths

Reply:
Sure. Fixed.

Reply:
Sure. Fixed.